### DISCUSSION TITLE
the Effect of perturbations on the Grace level1B data

### DISCUSSION THREAD

**Discussion by Maylisoo** [groops-devs/groops#87] [Q&A]
Hi 
I want to remove the perturbation effects (Solid earth tide, Ocean tide, Solid pole tide, Ocean pole tide, Nontidal atmosphere and ocean dealiasing, and Third-body perturbations from Sun, Moon, and planets) from the Grace level1b data. may it possible to use the GPS Navigation and KBR1B data of Grace(GNV1B and KBR1B) as an input file, compute the effect of perturbations, and subtract it from observations?  I want to know if I can use GROOPS Software for this purpose. 
 Thank you for your Support. 
Best Regards,
Maylisoo

**Comment by tmayerguerr** [comment]
Dear Maylisoo,
Please have a look at our cookbook [GRACE gravity field recovery](https://groops-devs.github.io/groops/html/cookbook.gravityFieldGrace.html). Follow the example until the Variational equations.
Afterwards you can use the programs [Variational2Orbit](https://groops-devs.github.io/groops/html/Variational2Orbit.html) and [SimulateSatelliteTracking](https://groops-devs.github.io/groops/html/SimulateSatelliteTracking.html).  You can compare the output of these programs with the original Grace data (e.g. with [InstrumentMultiplyAdd](https://groops-devs.github.io/groops/html/InstrumentMultiplyAdd.html)).
regards
Torsten
